zio-http/src/test/scala/zhttp/service/ServerSpec.scala

```diff
@@ -36,7 +36,13 @@ object ServerSpec extends HttpRunnableSpec {
     case _ -> !! / "HExitFailure" => Response.fromHttpError(HttpError.BadRequest())
   }
 
-  private val app = serve { nonZIO ++ staticApp ++ DynamicServer.app }
+  private val activeAllocations: ZIO[DynamicServer, Nothing, Unit] = for {
+    alloc <- DynamicServer.getStart.map(_.allocator.get)
+    ah    <- getActiveHeapBuffers(alloc)
+    ad    <- getActiveDirectBuffers(alloc)
+
+  } yield Console.println(s"Active heap buffers: $ah, Active direct buffers: $ad")
```

Comment:
We can keep a constant value to assert on, which should be equal to the number of tests that use `freeze`